Articles/Blog/2023-04-developer-surveys.md

```diff
@@ -8,109 +8,126 @@
-
-### What was involved in designing the survey?
-
-Designing the survey was a bit of a creative task, because I wanted to include questions that were relevant for lab developers, but still touch on expected content like favorite languages and editors.
-I spent a few weeks carefully going through public developer surveys that I knew about, along with previous application surveys at the lab.
-I decided on a set of categories that would be important for the lab (e.g., about Developer Experience, Developer, Happiness, Innovation, Collaboration, Cloud, to name a few) and then fine tuned questions within each category.
-While a lot of surveys don't like open answer questions because it's harder to throw into an Excel sheet and plot, I had an instinct that the best feedback might come from open response.
-I decided to take a balanced approach of using (for the most part) structured questions, but along with each section, giving an opportunity for open ended answers.
-While Likert scales could show a general sentiment, these open-ended responses would be where the heart of developer feedback came in.
-You really can't put people in a check box, nor can you ask them to express a full need or sentiment in one.
-
-Once I had created all of my questions, I put it into a Microsoft Form (what we have to use) and then showed to the RADIUSS team for feedback.
-As an aside, if you are able to use Google Forms I highly recommend this over what we had to use, because I found Microsoft Forms much harder to use.
-We had a few rounds of feedback, and also sent it out for some testing by a set of volunteers.
-As you might expect, many questions were refined, simplified, or entirely removed.
-Our perspective at this time was to do our best, and not worry if it wasn't perfect.
-We would learn from the experience and try differently the next time.
-A few months before the end of 2022, the survey was ready.
+I knew then that I wanted to continue my basic research to understand open source software and developer needs -- research I had started at Stanford University with my first [software survey](https://stanford-rc.github.io/stanford-software-survey/).
+At LLNL, there had previously been an annual survey of just the flagship simulation code teams that resulted in a dense, internally distributed document of survey results in text and plots.
+I saw a missed opportunity in that.
+
+I thought we might have more fun making an annual tradition of sending out a survey and collecting results (still internally) in a more interactive web interface giving the LLNL developer community yearly updates about what's going on in software development at LLNL.
+In addition, one code team in particular, the [RADIUSS](https://software.llnl.gov/radiuss/) team, served as a focal point for early work on the survey because the RADIUSS project involves a wide variety of interdependent LLNL software projects.
```

Comment:
```suggestion
In addition, one developer team in particular, the [RADIUSS](https://software.llnl.gov/radiuss/) team, served as a focal point for early work on the survey because the RADIUSS project supports a wide variety of interdependent LLNL software projects.
```
This isn't a code team.